specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/preview/2021-07-01-preview/galleryImages.json

```diff
@@ -0,0 +1,439 @@
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Lists all galleryimages under the subscription.",
+            "schema": {
+              "$ref": "#/definitions/galleryimagesListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "galleryimagesProperties": {
+      "type": "object",
+      "x-ms-client-flatten": true,
+      "properties": {
+        "containerName": {
+          "type": "string",
+          "description": "Container Name for storage container"
+        },
+        "imagePath": {
+          "type": "string",
+          "description": "location of the image the gallery image should be created from"
+        },
+        "osType": {
+          "type": "string",
+          "enum": [
+            "Windows",
+            "Linux"
+          ],
+          "x-ms-enum": {
+            "name": "OperatingSystemTypes",
+            "modelAsString": false
+          },
+          "description": "operating system type that the gallery image uses. Expected to be linux or windows"
+        },
+        "provisioningState": {
+          "type": "string",
+          "description": "Provisioning state of the gallery image."
+        },
+        "resourceName": {
+          "type": "string",
+          "description": "name of the object to be used in moc"
+        },
+        "status": {
+          "type": "object",
+          "description": "MOCGalleryImageStatus defines the observed state of MOCGalleryImage"
+        }
+      }
+    },
+    "galleryimages": {
+      "type": "object",
+      "x-ms-azure-resource": true,
+      "x-ms-client-flatten": true,
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/galleryimagesProperties"
+        },
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Resource Id"
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Resource Name"
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Resource Type"
+        },
+        "location": {
+          "type": "string",
+          "description": "The resource location",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData"
+        },
+        "extendedLocation": {
```

Comment:
>extendedLocation [](http://example.com/codeflow?start=9&length=16)

This belongs in the property bag.

Reply:
+1 this shouldn't be here

Reply:
Agreed. And, this should *not* be a breaking change. @Arkanayan ^

Reply:
Resolved.